06-regression.Rmd

```diff
@@ -416,6 +416,7 @@ Second, we take the saved model in `score_model` and apply the `get_regression_t
 
 ```{r moderndive-figure-wrapper, echo=FALSE, fig.align='center', fig.cap="The concept of a 'wrapper' function."}
 knitr::include_graphics("images/flowcharts/flowchart.011-cropped.png")
+knitr::include_graphics("images/ss/560016454.png")
```

Comment:
Photo looks good. Could you edit `images/flowcharts/flowchart.key` and output to `.png` in `images/flowcharts/flowchart.011-cropped.png` so that the input & output texts are there + arrows, so it looks like: 
<img width="857" alt="Screen Shot 2019-06-17 at 5 45 46 PM" src="https://user-images.githubusercontent.com/4574203/59638977-d2ecd800-9127-11e9-8993-713a8d9b212b.png">